modules/workload-identity/variables.tf

```diff
@@ -137,3 +137,9 @@ variable "gcp_sa_description" {
     error_message = "The Google service account description must be at most 256 characters"
   }
 }
+
+variable "create_ignore_already_exists" {
```

Comment:
Could we add `gcp_sa_` to the variable name similar to the existing variables?

Reply:
Totally forgot to look for naming conventions :+1: